src/app/app.module.ts

```diff
@@ -1,20 +1,84 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
 import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { NgModule } from '@angular/core';
+
+import { MatButtonModule } from '@angular/material/button';
```

Comment:
We could create a new module to export all material module. Here you have a example:

https://github.com/deviatan/horologium/blob/master/frontend/app/modules/material/material.module.ts

Reply:
As we agreed on the real world ™ maybe in the future